src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -78,7 +79,7 @@ const DisplayNote = React.forwardRef<HTMLElement, DisplayNoteProps>((
     </Dropdown>
     <CloseIcon onClick={onBlur} />
     <label>Note:</label>
-    <TruncatedText text={note} isFocused={isFocused} />
+    <TruncatedText text={note} isFocused={isFocused} onChange={() => setTextToggle(!textToggle)} />
```

Comment:
In this case, it doesn't matter because it is the only function which is modifying this state, but overall it's good practc to use previous state, I'll change this